.github/workflows/publish.yaml

```diff
@@ -1,4 +1,4 @@
-name: 'publish'
+na8me: 'publish'
```

Comment:
This looks like a typo to me

Reply:
Yep. Misclicked there somehow. Good catch!